source/_docs/installation/virtualenv.markdown

```diff
@@ -23,9 +30,31 @@ $ sudo apt-get install python3-pip python3-dev
 $ sudo pip3 install --upgrade virtualenv
 ```
 
-### {% linkable_title Step 1: Create a Home Assistant user & group %}
+## {% linkable_title Step 2: Setup virtualenv %}
+
+```bash
+$ cd $HOME
+$ mkdir homeassistant
+$ virtualenv -p python3 homeassistant
```

Comment:
I would prefer `$ python3 -m venv homeassistant`. As this is the same way it's described in the official Python documentation.